CMSIS/DSP/PythonWrapper/setup.py

```diff
@@ -14,7 +14,7 @@
   # since the visual compiler and the win platform are
   # not supported by default in arm_math.h
 else:
-  cflags = ["-Wno-unused-variable","-Wno-implicit-function-declaration",config.cflags]
+  cflags = ["-Wno-unused-variable","-Wno-implicit-function-declaration",config.cflags,"-D__GNUC_PYTHON__"]
```

Comment:
If setup.py is supporting windows and Linux then I think some logic should be added to detect it and set __GNUC_PYTHON__ only when building on Linux.

Otherwise it may break the build with MSVC.

Reply:
@christophe0606  the cflags being set here are specifcally for linux. The first part of the if statement is doing the windows check.

Reply:
@cdknorow Awesome. Thanks.